### PR TITLE
fix(AAPD-825): fix page titles

### PR DESCRIPTION
**Pull request by Si3rr4wow**
## Ticket Number

https://pins-ds.atlassian.net/browse/AAPD-

## Checklist

- [ ] Requires infrastructure changes
- [ ] I have updated the documentation accordingly
- [ ] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `main` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: packages/web-comment/src/routes/comment-appeal/find-planning-appeal/index.njk [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {% extends "../../../index.njk" %}
-{% set title = "Enter appeal reference" %}
+{% set title = "Enter the appeal reference - Comment on a planning appeal - Welcome to GOV.UK" %}
```
Comment (beejjacobs) [inline packages/web-comment/src/routes/comment-appeal/find-planning-appeal/index.njk:2]: Is this suffix needed for every page?

**Review by beejjacobs — APPROVED** [review]
LGTM

File: packages/web-comment/src/routes/comment-appeal/find-planning-appeal/index.njk (hunk 1 above)
Comment (Si3rr4wow) [reply packages/web-comment/src/routes/comment-appeal/find-planning-appeal/index.njk:2]: Good point, it seems like it probably should be but this is the only page Mo picked this defect up on. Do you know a smart nunjucksian way to put a postfix on titles?

Comment (beejjacobs) [reply packages/web-comment/src/routes/comment-appeal/find-planning-appeal/index.njk:2]: Honestly I'm a bit confused how this works at all - the govuk root template appears to use `pageTitle`. 

I guess if our index/main template were to set `pageTitle` by concatenating `title` and the `Comment on a ...` suffix?

Comment (beejjacobs) [reply packages/web-comment/src/routes/comment-appeal/find-planning-appeal/index.njk:2]: Looks like this was picked up on other tickets too, just not mentioned outside of the word doc attachment: https://pins-ds.atlassian.net/browse/AAPD-836

@Si3rr4wow are you happy to sort for all pages?

File: packages/web-comment/src/routes/decided-appeals/index.njk [hunk 2]
```diff
@@ -1,33 +1,33 @@
 {% extends "../../index.njk" %}
-{% set title = "Enter appeal reference" %}
```
Comment (beejjacobs) [inline packages/web-comment/src/routes/decided-appeals/index.njk:2]: I think based on this prototype: https://appeals-prototype.herokuapp.com/interested/find-appeal/v9/closed-appeals?status=completed&appeal=1&appealCode=010234

The title is just meant to be "Decided appeals - ..."

**Review by beejjacobs — COMMENTED** [review]
Solution looks good: one query on content.

File: packages/web-comment/src/routes/decided-appeals/index.njk [hunk 3]
```diff
@@ -1,33 +1,33 @@
...
+{% set title = [decidedAppeals.length, " decided Appeals found ", postcode] | join %}
```
Comment (beejjacobs) [inline packages/web-comment/src/routes/decided-appeals/index.njk:2]: I think based on this prototype: https://appeals-prototype.herokuapp.com/interested/find-appeal/v9/closed-appeals?status=completed&appeal=1&appealCode=010234

The title is just meant to be "Decided appeals - ..."

Comment (Si3rr4wow) [reply packages/web-comment/src/routes/decided-appeals/index.njk:2]: Good spot, amended that now :)

**Review by beejjacobs — APPROVED** [review]
LGTM!

**Pull request merged**